turbinia/lib/libcloudforensics.py

```diff
@@ -293,17 +293,8 @@ def get_or_create_analysis_vm(self, vm_name, boot_disk_size, cpu_cores=4):
 
     # Analysis software to install.
     packages_to_install = [
-        'python-plaso',
-        'xmount',
-        'sleuthkit',
-        'libfvde-tools',
-        'libbde-tools',
-        'plaso-tools',
-        'jq',
-        'ncdu',
-        'htop',
-        'binutils',
-        'upx-ucl'
+        'python-plaso', 'xmount', 'sleuthkit', 'libfvde-tools', 'libbde-tools',
```

Comment:
I wonder how it decides when to have an entry per line.  Generally I prefer for data structures to be one entry per line, and for function calls to be multiple per line.